client/state/data-layer/wpcom/sites/comment-counts/index.js

```diff
@@ -0,0 +1,71 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+// import { translate } from 'i18n-calypso';
+// import { flatMap, flatten, isArray, map } from 'lodash';
```

Comment:
Forgot to clean this up? :)

Reply:
Thanks, I can spin that up in another PR